chromadb/test/property/test_cross_version_persist.py

```diff
@@ -256,18 +256,28 @@ def persist_generated_data_with_old_version(
             # In order to test old versions, we can't rely on the not_implemented function
             embedding_function=not_implemented_ef(),
         )
-        coll.add(**embeddings_strategy)
+        result = coll.add(**embeddings_strategy)
+
+        if embeddings_strategy["ids"] is None:
```

Comment:
We should gate this on the version, similar to what we do with create collection above.

Reply:
Should rename to match `RecordSet` type.

Reply:
done